LoRaEngine/README.md

```diff
@@ -2,28 +2,21 @@
 
 A **.NET Standard 2.0** solution with the following projects:
 
-- **PacketForwarderHost** - executable and IoTEdge configuration files.
-- **UDPListener** - executable
-- **LoRaTools** - library
-- **LoRaServer** - IoT edge module executable, Dockerfile, etc.
-- **DevTools** - submodule folder, check it out using the command
-  `git submodule update --init --recursive` and add the projects to the visual studio solution
-  - **PacketForwarderSimulator** - executable
-  - **DevTool1** - executable used during development process
-  - **DevTool2** - executable used during development process
-  - etc . . .
-
-**NOTE:** Until we have a unit test framework in place, we are relying on small executables, maintained by each developer, for the purposes of testing and demonstrating module functionality. Each of these executables should have a README.md file that demonstrates how to run the code.
+- **modules** - Azure IoT Edge modules.
+  - **LoRaWanPktFwdModule** packages the network forwarder into an IoT Edge compatible docker container. See https://github.com/Lora-net/packet_forwarder and https://github.com/Lora-net/lora_gateway.
+  - **LoRaWanNetworkSrvModule** - is the LoRaWAN network server implementation.
+- **LoraKeysManagerFacade** - An Azure function handling device provisioning (e.g. LoRa network join, OTAA) with Azure IoT Hub as persistence layer.
+- **LoRaDevTools** - library for dev tools
```

Comment:
mention the fact that it is a submodule? (hence won't be checked by default). they need to run git submodule update